hazelcast/src/main/java/com/hazelcast/nio/serialization/ClassDefinitionBuilder.java

```diff
@@ -70,6 +68,15 @@ public ClassDefinitionBuilder(int factoryId, int classId, int version) {
         this.version = version;
     }
 
+    /**
+     * IMPORTANT: Make sure that the version matches the portableVersion in the SerializationService
+     *
```

Comment:
add `@param` javadoc for completeness

Reply:
I followed a better approach: removed all `@param X X to use` from other constructors.